src/containers/org/OrgDataSetContainer.js

```diff
@@ -74,6 +75,12 @@ const CountBadge = styled(Badge)`
   margin-right: 5px;
 `;
 
+const StyledChip = styled(Chip)`
+  color: ${NEUTRAL.N700};
+  background: ${NEUTRAL.N100};
+  font-weight: 600;
+`;
```

Comment:
should we just switch the default chip to match these colors?

Reply:
hmm dunno the designs are in consistent in that regard.

Reply:
I agree, we should use the chip colors. I've told Chris to standardize around that.